rst_files/orth_proj.rst

```diff
@@ -919,6 +938,12 @@ basis by QR decomposition and project once more.
 
 .. code-block:: julia
 
-    Py2 ≈ Q * Q' * y
+    Py3 = Q * Q' * y
+
+.. code-block:: julia 
+    :class: test 
+    @testset "Test Py3" begin 
+        @test norm(Py3-Py1) ≈ 0 atol = 1e-10
```

Comment:
Wouldn't this just be `@test Py1 ≈ Py3 atol = 1e-10`?

Reply:
Maybe they're equivalent, but that was the quantity I used in the REPL.

Reply:
We probably want to test `a  ≈ b` since there might be lectures that don't use `LinearAlgebra` and we don't want to include that dependency as a `target`.

Reply:
That's a good point. Will do this as well.